example/settings.py

```diff
@@ -65,7 +65,7 @@
     'two_factor',
     'example',
 
-    'debug_toolbar',
+    # 'debug_toolbar',
```

Comment:
With version 1.6 of Django Debug Toolbar that should be resolved, can you verify that you're on that version?

Anyway I think its best to separate it from this PR.